pyanalyze/test_annotations.py

```diff
@@ -127,7 +127,7 @@ def capybara(x: X, y: Y) -> None:
     def test_literal(self):
         from typing_extensions import Literal
 
-        def capybara(x: Literal[True], y: Literal[True, False]) -> None:
+        def capybara(x: Literal[True], y: Literal[(True, False)]) -> None:
```

Comment:
I don't think pyanalyze will catch this as the AST is the same either way. Mypy accepts this too.

Reply:
Evidence: https://mypy-play.net/?mypy=latest&python=3.10&gist=0df0421d5c85f3b75f65a51cae8616ce

Reply:
Thanks for checking then I think I can close this, and open an issue in Python repo.